streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -88,6 +88,58 @@ class StreamingListenerSuite extends TestSuiteBase with Matchers {
     }
   }
 
+  test("SPARK-6766: batch info should be submitted") {
```

Comment:
Hmm, each of the two new unit tests is taking .7 seconds. Not good. Best to merge them into the first unit test. 
The reason its taking time is that its starting a new SparkContext every time, and running batches after that.